README.md

```diff
@@ -119,7 +155,7 @@ Add this plugin to your pom.xml:
               <dependency>
                   <groupId>org.cqfn.diktat</groupId>
                   <artifactId>diktat-rules</artifactId>
-                  <version>0.1.1</version> <!-- replace it with diktat latest version -->
+                  <version>0.1.3</version> <!-- replace it with diktat latest version -->
```

Comment:
will you add auto update here in your update script?

Reply:
Readme is handled by update script, I think we forgot to increment this version once and it was not checked since than. Script only changes latest version to the new value, so it should start getting updates now.